src/EditorFeatures/CSharpTest/CSharpEditorServicesTest.csproj

```diff
@@ -205,6 +205,7 @@
     <Compile Include="Diagnostics\UseImplicitOrExplicitType\UseExplicitTypeTests_FixAllTests.cs" />
     <Compile Include="Diagnostics\UseImplicitOrExplicitType\UseImplicitTypeTests_FixAllTests.cs" />
     <Compile Include="Diagnostics\UseImplicitOrExplicitType\UseImplicitTypeTests.cs" />
+    <Compile Include="DocumentationComments\CodeFixes\AddDocCommentParamNodeCodeFixProviderTests.cs" />
```

Comment:
i feel like this should be AddDocumentNode... to match RemoveDocCommentNode.  The usage of 'param' in one and not the other seems off.